skipchain/skipchain.go

```diff
@@ -107,6 +112,15 @@ func (s *Service) StoreSkipBlock(psbd *StoreSkipBlock) (*StoreSkipBlockReply, on
 			return nil, onet.NewClientErrorCode(ErrorBlockContent,
 				"We're not responsible for latest block")
 		}
+		if len(prev.ForwardLink) > 0 {
+			return nil, onet.NewClientErrorCode(ErrorBlockContent,
+				"the latest block already has a follower")
+		}
+		if !s.newBlockStart(prev) {
+			return nil, onet.NewClientErrorCode(ErrorBlockInProgress,
+				"this skipchain-id is currently processing a block")
+		}
+		defer s.newBlockEnd(prev)
```

Comment:
It's line 119 - it checks that the leader is not adding another block to the `prev`, which is the latest ackknowledged block.